src/web/components/App.tsx

```diff
@@ -390,6 +392,30 @@ export const App = ({ CAPI, NAV }: Props) => {
 					/>
 				</HydrateOnce>
 			))}
+			{CAPI.quizAtoms.map((quizAtoms, index) => (
+				<HydrateOnce
+					key={index}
+					root="quiz-atom"
+					index={quizAtoms.quizIndex}
+				>
+					<>
+						{quizAtoms.quizType === 'personality' &&
+							quizAtoms.resultBuckets && (
```

Comment:
I think when we discussed this with Jon we spoke about how it's better to render the quiz even without results, rather than rendering nothing at all. In both cases the quiz is 'broken' because the authors have failed too provide the answers but still rendering the quiz feels like a more graceful error state?

It's also more semantic I think to handle the lack of results inside the component rather than here, in DCR

Reply:
The JSON will always return `resultBuckets` even if the quiz is not a personality quiz. I have changed for this prop to not be optional